packages/editor/src/components/post-taxonomies/hierarchical-term-selector.js

```diff
@@ -319,15 +318,16 @@ class HierarchicalTermSelector extends Component {
 			const id = `editor-post-taxonomies-hierarchical-term-${ term.id }`;
 			return (
 				<div key={ term.id } className="editor-post-taxonomies__hierarchical-terms-choice">
-					<input
+					<CheckboxControl
 						id={ id }
```

Comment:
This doesn't need an `id`.

Reply:
it does for the label for attribute :)

Reply:
Look at the implementation of the component. It uses `withInstanceId`.

Reply:
yes I noticed and removed my initial comment :D

Reply:
:smile: